pkgs/racket-doc/scribblings/reference/for.scrbl

```diff
@@ -111,6 +113,15 @@ binding and expansion. Support for @racket[#:splice] clauses is
 intended less for direct use in source @racket[for] forms than for
 building new forms that expand to @racket[for].
 
+At @racket[#:on-length-mismatch mismatch-expr] clause is similar to
```

Comment:
Change to `A @racket[#:on-length-mismatch mismatch-expr] clause`

Reply:
Fixed, but I went with `An`.